README.md

```diff
@@ -0,0 +1,42 @@
+# Avidemux
+
+Avidemux is a simple cross-platform video editor for Linux, Windows and MacOsX.
+
+# Build Instructions
+
+Get the main repository:
```

Comment:
This instruction is probably not needed, whoever has reached this point either has the source or the basic skills.

Reply:
I wanted to, that whoever has reached this point, can build-and-run regardless of skills.